docs/guides/bearer_token/bearer_token_guide.md

```diff
@@ -0,0 +1,68 @@
+---
+uid: Guides.BearerToken
+title: Working with Bearer token
+---
+
+# Working with Bearer token
+
+Some endpoints in Discord API require a Bearer token, which can be obtained through [OAuth2 flow](https://discord.com/developers/docs/topics/oauth2). Discord.Net allows you to interact with these endpoints using the [DiscordRestClient].
+
+## Initializing a new instance of the client
+[!code-csharp[Initialize DiscordRestClient](samples/rest_client_init.cs)]
+
+## Getting current user
+
+The [DiscordRestClient] gets the current user when `LoginAsync()` is called. The user object can be found in the `CurrentUser` property.
+
+If you need to fetch the user again, the `GetGetCurrentUserAsync()` method can be used.
```

Comment:
`GetGetCurrentUserAsync` seems should be `GetCurrentUserAsync`

Reply:
Oh, nice catch, thanks!